File: backup/Twitch/app/src/main/java/com/exact/twitch/ui/pagers/GamePagerFragment.java
package com.exact.twitch.ui.pagers;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.view.View;

public class GamePagerFragment extends MediaPagerFragment {

    public static GamePagerFragment newInstance(String game) {
        Bundle args = new Bundle();
        args.putString("game", game);
        GamePagerFragment fragment = new GamePagerFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        setAdapter(new GamePagerAdapter(requireActivity(), getChildFragmentManager(), getArguments()));
    }
}

File: app/src/main/java/com/exact/twitch/ui/pagers/GamePagerFragment.kt
package com.exact.twitch.ui.pagers

import android.os.Bundle
import android.view.View
import androidx.core.os.bundleOf
import com.exact.twitch.model.game.Game

class GamePagerFragment : MediaPagerFragment() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setAdapter(GamePagerAdapter(requireActivity(), childFragmentManager, arguments!!))
    }

    companion object {
        fun newInstance(game: Game) = GamePagerFragment().apply { arguments = bundleOf("game" to game) }
    }
}
